src/mavo.js

```diff
@@ -177,7 +177,18 @@ var _ = self.Mavo = $.Class({
 
 		this.permissions.can("login", () => {
 			// We also support a URL param to trigger login, in case the user doesn't want visible login UI
-			if (Mavo.Functions.url("login") !== null && this.index == 1 || Mavo.Functions.url(this.id + "-login") !== null) {
+			let loginUrlParam = null;
+			if (Mavo.Functions.url("login") !== null && this.index === 1) {
+				loginUrlParam = "login";
+			}
+			else if (Mavo.Functions.url(this.id + "-login") !== null) {
+				loginUrlParam = this.id + "-login";
+			}
+			if (loginUrlParam !== null) {
```

Comment:
Did eslint not complain about the lack of empty line before the if block? 
Currently it looks like it's part of the previous conditional. An empty line before and after control flow blocks helps visual grouping.

Reply:
Did not get an eslint warning on this

Reply:
Are you sure ESLint is running? [The rule is there](https://github.com/mavoweb/mavo/blob/master/.eslintrc.json#L31)